AUTHORS

```diff
@@ -1 +1,2 @@
 Andy Williams <andy@fyne.io>
+Tai Groot <tai@taigrr.com>
```

Comment:
Sorry to ask but could you remove this for now? We generally hold back AUTHORS list for larger additions like features, design review and so on.